giddy-banana/routes/articleRoute.js

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import { getArticles, postArticles, deleteArticles, updateArticles } from "../controllers/articleController.js";
+
+const articlesRouter = new express.Router();
+articlesRouter.use(express.json());
```

Comment:
Pretty sure that's not needed

Reply:
But I might be wrong here. Need to look at the articles routers. Just seems weird because json reading should only be required for the post endpoint.